app/views/shared/views/_show_topology.html.haml

```diff
@@ -0,0 +1,77 @@
+- tooltipOptions = {"tooltip-placement" => "bottom", "tooltip" => "{{legendTooltip}}"}
```

Comment:
> Its shared between the container_topology tab and the topology view in ems_container

if we would need another "shared" topology for middleware and network? where would you place it and how would you call that file if this one is in shared folder and is called the general "show...." ?